front/lib/api/assistant/agent.ts

```diff
@@ -359,6 +360,12 @@ export async function* runMultiActionsAgent(
       }
 
       specifications.push(r.value);
+    } else if (isWebsearchConfiguration(a)) {
+      specifications.push({
+        name: a.name ?? "websearch",
+        description: a.description ?? "Websearch",
```

Comment:
This description is wrong?

Reply:
Changed to use generateWebsearchSpeciciation